### PR TITLE
Astro 8/22 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/deployment-metrics.md [hunk 1]
```diff
@@ -154,36 +154,43 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
 
 :::info
 
 Astro does not export any [service checks](https://docs.datadoghq.com/integrations/airflow/?tab=host#service-checks) to Datadog. Information about the general health of your Deployment is available only as part of the Cloud UI's [Deployment health](#deployment-health) metric.
 
 :::
 
+#### Prerequisites
+
+- Your Deployment must be running Astro Runtime 9. See [Upgrade Astro Runtime](upgrade-runtime.md).
```
Comment (melugoyal) [inline astro/deployment-metrics.md:169]: runtime 9 isn't needed for metrics forwarding, only for logs

File: astro/view-logs.md [hunk 2]
```diff
@@ -119,10 +118,39 @@ To continue monitoring logs, run `astro dev logs --follow`. The `--follow` flag
...
+  :::info
+
+  By default, the Astro Datadog integration also sends a Deployment's [Airflow metrics](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html) to Datadog. To send only task logs to Datadog, additionally set the following environment variable:
+
+  - **Key**: `ASTRO_DATADOG_METRICS_DISABLED`
+  - **Value**: `true`
+
+  :::
+
+3. (Optional) Set the following [environment variable](environment-variables.md) on your Deployment to send your metrics to a specific [Datadog site](https://docs.datadoghq.com/getting_started/site/):
```
Comment (melugoyal) [inline astro/view-logs.md:149]: we can also document the optional config of `ASTRO_DATADOG_TASK_LOGS_TAGS`, which has the same behavior as `AIRFLOW__METRICS__STATSD_DATADOG_TAGS`, but just for additional logging tags

File: astro/deployment-metrics.md [hunk 3]
```diff
@@ -154,36 +154,43 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
+
```
Comment (jwitz) [inline astro/deployment-metrics.md:170]:
```suggestion
```

File: astro/view-logs.md [hunk 4]
```diff
@@ -119,10 +118,39 @@ To continue monitoring logs, run `astro dev logs --follow`. The `--follow` flag
...
+
+    - **Key**: `DATADOG_SITE`
+    - **Value**: Your Datadog site name. For example, `US3`.
+
+4. Click **Save variable**.
```
Comment (jwitz) [inline astro/view-logs.md:154]:
```suggestion
4. (Optional) Add the following environment variables to create [custom Datadog tags](https://docs.datadoghq.com/getting_started/tagging/) associated with your Deployment:

   - **Key ** `ASTRO_DATADOG_TASK_LOGS_TAGS`
   - **Value ** `<tag-key-1>:<tag-value-1>,<tag-key-2>:<tag-value-2>`
   
5. Click **Save variable**.
```

File: astro/deployment-metrics.md [hunk 5]
```diff
@@ -154,36 +154,43 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
+   - **Key 1** `AIRFLOW__METRICS__STATSD_DATADOG_ENABLED`
+   - **Value 1** `True`
 
-  :::info
-  
-  If you're configuring custom Datadog tags for a Deployment using Astro Runtime 5 or earlier, you must additionally add `datadog` to your Astro project `requirements.txt` file.
-  
-  :::
+   - **Key 2** `AIRFLOW__METRICS__STATSD_DATADOG_TAGS`
+   - **Value 2** `<key-1>:<value-1>,<key-2>:<value-2>`
```
Comment (jwitz) [inline astro/deployment-metrics.md:193]:
```suggestion
   - **Value 2** `<tag-key-1>:<tag-value-1>,<tag-key-2>:<tag-value-2>`
```

File: astro/deployment-metrics.md [hunk 6]
```diff
@@ -154,36 +154,39 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
 
 Astro does not export any [service checks](https://docs.datadoghq.com/integrations/airflow/?tab=host#service-checks) to Datadog. Information about the general health of your Deployment is available only as part of the Cloud UI's [Deployment health](#deployment-health) metric.
 
 :::
 
+#### Setup
+
 1. Create a new Datadog API key or copy an existing API key. See [API and Application Keys](https://docs.datadoghq.com/account_management/api-app-keys/).
 2. In the Cloud UI, select a Workspace and then select an Astro Deployment for which you want to export metrics.
 3. Create a new [environment variable](environment-variables.md#set-environment-variables-in-the-cloud-ui) in your Deployment with the Datadog API key from step 1:
```
Comment (lzdanski) [inline astro/deployment-metrics.md:171]: The formatting  with bullet points on this page looks a little odd depending on the size of the browser window, and might be a docusaurus thing - a fix is probably outside the scope on this and might be out of our control, but wanted to show you. For example, these unordered lists are spaced the same way, with no line breaks between them in the source, but are rendered differently.
Screenshot 2023-08-22 at 9 44 04 AM

And in a smaller window, the line break made this huge whitespace.
Screenshot 2023-08-22 at 9 41 35 AM

File: astro/view-logs.md [hunk 7]
```diff
@@ -119,10 +118,44 @@ To continue monitoring logs, run `astro dev logs --follow`. The `--follow` flag
...
+    Select the **Secret?** checkbox for `DATADOG_API_KEY`. This ensures that your Datadog API key is saved securely and is not available to Workspace users in plain text.
+
+  :::info
+
+  By default, the Astro Datadog integration also sends a Deployment's [Airflow metrics](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html) to Datadog. To send only task logs to Datadog, additionally set the following environment variable:
+
+  - **Key**: `ASTRO_DATADOG_METRICS_DISABLED`
+  - **Value**: `true`
+
+  :::
```
Comment (lzdanski) [inline astro/view-logs.md:147]: This note isn't formatting correctly due to the bullet points nesting level. They might need the full tab in - this change has two spaces.
```suggestion
  :::info

  By default, the Astro Datadog integration also sends a Deployment's [Airflow metrics](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html) to Datadog. To send only task logs to Datadog, additionally set the following environment variable:

    - **Key**: `ASTRO_DATADOG_METRICS_DISABLED`
    - **Value**: `true`

  :::
```

**Review by lzdanski — COMMENTED** [review]
Overall this LGTM, there's one formatting thing that docusaurus is doing a little strangely and then a typo I found.

File: astro/view-logs.md [hunk 8]
```diff
@@ -119,10 +118,44 @@ To continue monitoring logs, run `astro dev logs --follow`. The `--follow` flag
 
 ## Export task logs to Datadog (_AWS only_)
```
Comment (melugoyal) [inline astro/view-logs.md:119]: cool so presumably once the next runtime minor version is released with datadog logging support for Azure and GCP, we'll just update this to remove "AWS only"?

**Review by melugoyal — APPROVED** [review]

File: astro/deployment-metrics.md [hunk 9]
```diff
@@ -154,36 +154,39 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
-
-   - **Key** `AIRFLOW__METRICS__STATSD_DATADOG_ENABLED`
-   - **Value** `True`
+  
+5. (Optional) Add the following environment variables to create [custom Datadog tags](https://docs.datadoghq.com/getting_started/tagging/) associated with your Deployment:
 
-   - **Key** `AIRFLOW__METRICS__STATSD_DATADOG_TAGS`
-   - **Value** `<key-1>:<value-1>,<key-2>:<value-2>`
+   - **Key 1** `AIRFLOW__METRICS__STATSD_DATADOG_ENABLED`
+   - **Value 1** `True`
```
Comment (jwitz) [inline astro/deployment-metrics.md:186]:
```suggestion
   - **Key 1**: `AIRFLOW__METRICS__STATSD_DATADOG_ENABLED`
   - **Value 1**: `True`
```

File: astro/deployment-metrics.md [hunk 10]
```diff
@@ -154,36 +154,39 @@ The **Worker CPU** and **Worker Memory** charts in the Cloud UI provide visibili
...
 
-  :::info
-  
-  If you're configuring custom Datadog tags for a Deployment using Astro Runtime 5 or earlier, you must additionally add `datadog` to your Astro project `requirements.txt` file.
-  
-  :::
+   - **Key 2** `AIRFLOW__METRICS__STATSD_DATADOG_TAGS`
+   - **Value 2** `<tag-key-1>:<tag-value-1>,<tag-key-2>:<tag-value-2>`
```
Comment (jwitz) [inline astro/deployment-metrics.md:189]:
```suggestion
  
    - **Key 2**: `AIRFLOW__METRICS__STATSD_DATADOG_TAGS`
   - **Value 2**: `<tag-key-1>:<tag-value-1>,<tag-key-2>:<tag-value-2>`
```

File: astro/view-logs.md (hunk 8 above)
Comment (jwitz) [reply astro/view-logs.md:119]: Yep!

**Pull request merged**